resource/js/crowi.js

```diff
@@ -320,7 +320,7 @@ $(function() {
     var shortPath = $link.data('short-path');
```

Comment:
I think, to fix this problem, `shortPath` should be converted the return value of `data()` to string here.